plugin.php

```diff
@@ -115,6 +113,11 @@ function create_initial_json_routes() {
 		}
 
 		$controller->register_routes();
+
+		if ( post_type_supports( $post_type->name, 'custom-fields' ) ) {
+			$meta_controller = new WP_JSON_Meta_Posts_Controller( $post_type->json_base );
```

Comment:
I think we can just register by default for posts, users and comments, and figure out the abstraction in a later beta.